package.json

```diff
@@ -7,7 +7,8 @@
     "cypress": "^8.5.0"
   },
   "devDependencies": {
-    "cypress-file-upload": "^5.0.8"
+    "cypress-file-upload": "^5.0.8",
+    "faker": "^5.5.3"
```

Comment:
This was not even supposed to be tested - file upload, and doing stuff with faker, although I really do appreciate the effort. However, please do remember to be kind to yourself, and do not waste your time and energy where it is not necessary, it will bring you much joy in the future.